index.html

```diff
@@ -0,0 +1,624 @@
+                        <div class="card-heading">
+                            <img src="./images/program_icon_04.png" alt="" />
+                            <h3>cc exhibition</h3>
+                        </div>
+
+                        <div class="card-para">
+                            <p>
+                                We have time to share our thoughts and opinions with experts by
+                                topic.
+                            </p>
+                        </div>
+                    </div>
+
+                    <div class="card-contents">
+                        <div class="card-heading">
+                            <img src="./images/program_icon_04.png" alt="" />
+                            <h3>forum</h3>
+                        </div>
+
+                        <div class="card-para">
+                            <p>
+                                We have time to share our thoughts and opinions with experts by
+                                topic.
+                            </p>
+                        </div>
+                    </div>
+
+                    <div class="card-contents">
+                        <div class="card-heading">
+                            <img src="./images/program_icon_04.png" alt="" />
+                            <h3>Workshop</h3>
+                        </div>
+
+                        <div class="card-para">
+                            <p>
+                                We have time to share our thoughts and opinions with experts by
+                                topic.
+                            </p>
+                        </div>
+                    </div>
+
+                    <div class="card-contents">
+                        <div class="card-heading">
+                            <img src="./images/program_icon_05.png" alt="" />
+                            <h3>cc party</h3>
+                        </div>
+
+                        <div class="card-para">
+                            <p>
+                                We have time to share our thoughts and opinions with experts by
+                                topic.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-1"></div>
+                    <div class="col-10 join_summit">
+                        <center>Join summit 2023</center>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="row down_part">
+            <div class="col-12">
+                <center>
+                    <span>
+                        <a href="##">SEE THE WHOLE PROGRAM </a>
+                    </span>
+                </center>
+            </div>
+        </div>
+    </section>
+
+    <!--Down section-->
+    <section class="container down_section" id="desktop_speakers">
```

Comment:
- [ ] For the `speaker section ` the content should be left align instead of center align. Please make sure to make your content align to the left as well.

Design Guidelines:
<img width="416" alt="Screenshot 2023-03-24 at 11 05 34 PM" src="https://user-images.githubusercontent.com/44798044/227616563-b2fe655b-fb33-4653-8151-7a9eb39449ec.png">


Your Design:

<img width="568" alt="Screenshot 2023-03-24 at 11 05 41 PM" src="https://user-images.githubusercontent.com/44798044/227616469-192e02d3-14d3-44d5-8420-6dbd54af2690.png">